Testing/SystemTests/tests/analysis/reference/II.IRISFuryFitMulti.nxs.md5

```diff
@@ -1 +1 @@
-ce0d536337942b5662b8ecd107a5c71f
+d3c125646057bc0d1291c3286ddfab82
```

Comment:
what is the difference between these two files? Is it just the parameters being copied correctly?

Reply:
There was one number different

Reply:
was it by much?

Reply:
Less than 0.1%